jobs.php

```diff
@@ -271,6 +271,7 @@
         // Job start time, end time and elapsed time
         $start_time = $job['starttime'];
         $end_time   = $job['endtime'];
+
```

Comment:
I guess this is from your code.
There is another thing that I noticed now:

This line should be:
 "$job['endtime']     = CUtils::format_DateTime($end_time/* not $start_time*/, $config['datetime_format']);

I have a fork of webacula with lot of modifications in layout:
If you want to take a look....
https://github.com/wanderleihuttel/webacula/releases/tag/7.5.0


Best Regards
Wanderlei